src/Microsoft.AspNet.Mvc.Core/Builder/RouteBuilderExtensions.cs

```diff
@@ -0,0 +1,136 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Routing;
+
+namespace Microsoft.AspNet.Builder
+{
+    /// <summary>
+    /// Extension methods for <see cref="IRouteBuilder"/>.
+    /// </summary>
+    public static class RouteBuilderExtensions
```

Comment:
Would like to rename this to something like `MvcAreaRouteBuilderExtensions` so it's pretty specific (because it's in a generic namespace). Or can remove `Area` from the name if we might add more non-area stuff here.